app/views/entities/index.html.erb

```diff
@@ -1,22 +1,23 @@
-<div class="grid">
+<div class="flex items-center gap-44 p-5">
   <span>
-    <%= link_to 'Back' , groups_path, class: 'bg-second py-1 px-2 rounded font-semibold text-white' %>
+  <%= link_to groups_path do %>
+    <%= image_tag('back.png', class: 'w-6') %>
+  <% end %>
   </span>
-  <span class="text-2xl font-bold text-text justify-self-center self-center"> Transactions
+  <span class=""> Transactions
   </span>
 </div>
 <% @entities.each do |entity| %>
-  <div class="flex justify-between items-center">
-    <div class="flex items-center gap-3 my-3">
-      <span class="text-xl font-semibold"><%= entity.name %></span>
+  <div class="flex justify-between items-center py-3 px-5 my-2 mx-3 bg-white drop-shadow-md">
+    <div class="">
+      <span class=""><%= entity.name %></span>
       <span>$<%= entity.amount%></span>
     </div>
```

Comment:
# Hi can you follow this requirement below to complete your App? I advise you to work on the 3rd point below:

- [ ] "Add a new transaction" page

The user fills out a form to create a new transaction with:

- 1 name (mandatory)

- 2 amount (mandatory)

- 3 👉 categories (mandatory at least one) 👈 

- **Here is your App.** 



![6](https://github.com/BranBayou/my-budget/assets/63972670/c0599d1a-a159-426c-9cba-124f6a84407f)

NB: The comment above is part of the project requirement.